config/src/utils/jsonParser.lua

```diff
@@ -0,0 +1,45 @@
+-- Copyright 2018 Yahoo Inc.
+-- Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+
+local Json = require("lib.json")
+
+--- a module provides functions to parse json file or object.
+-- @module jsonParser
+
+local JsonTable = {}
+
+--- Save a lua table to a json file.
+--
+-- @param filepath  A file for saving json
+-- @param table A lua table to be saved as json file
+-- @return the lua table
+JsonTable.save = function(filepath, table)
+    local file = io.open(filepath, "w")
```

Comment:
When `io.open` fails the first parameter is `nil`, and it *also* returns a
second parameter, which is some sort of error message. At the minimum, we
should print out that error message if we fail to open the file.

Though we really should use a
[logging framework](https://github.com/rxi/log.lua), and log it.